users/manna-harbour_miryoku/rules.mk

```
@@ -1,5 +1,5 @@
-
-# generated from users/manna-harbour_miryoku/miryoku.org
+# generated from users/manna-harbour_miryoku/miryoku.org  -*- buffer-read-only: t -*-
 
 MOUSEKEY_ENABLE = yes        # Mouse keys(+4700)
 EXTRAKEY_ENABLE = yes        # Audio control and System control(+450)
+EXTRAFLAGS += -flto	     # Link Time Optimization to reduce code size, 31358->28034/28672
```

```suggestion
LTO_ENABLE = yes
```

This enables the same option, and disables action_macros, and action_functions, both of which seem to break when LTO is enabled.   And this is a lot simpler to use.